tracing/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  mongo:
+    image: mongo
+    restart: always
+    ports:
+      - 27017:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: insecure
+
+  mongo-express:
+    image: mongo-express
+    restart: always
+    ports:
+      - 8081:8081
+    environment:
+      ME_CONFIG_MONGODB_ADMINUSERNAME: root
+      ME_CONFIG_MONGODB_ADMINPASSWORD: example
+      ME_CONFIG_MONGODB_URL: mongodb://root:insecure@mongo:27017/
+      ME_CONFIG_BASICAUTH: false
```

Comment:
the docker compose on my machine complained about this and I had to change it to "false" (with quotes)

Reply:
Added quotes.